GetAthleteData.py

```diff
@@ -53,9 +57,12 @@ def computePrice(athlete_data):
         rushingYards = athlete['RushingYards']
         OffensiveSnapsPlayed = athlete['OffensiveSnapsPlayed']
         DefensiveSnapsPlayed = athlete['DefensiveSnapsPlayed']
-        # rushingTouch = athlete['RushingTouch']
+        Touchdowns = athlete['Touchdowns']
+        PassingInterceptions = athlete['PassingInterceptions']
+        FumblesLost = athlete['FumblesLost']
+        RushingTouchdowns = athlete['RushingTouchdowns']
 
-        sock.sendall((f'nfl,name={name},id={id},team={team},position={position} passingYards={passingYards},passingTouchdowns={passingTouch},reception={reception},receiveYards={receiveYards},receiveTouch={receiveTouch},rushingYards={rushingYards},OffensiveSnapsPlayed={OffensiveSnapsPlayed},DefensiveSnapsPlayed={DefensiveSnapsPlayed},price={price}\n').encode())
+        sock.sendall((f'nfl,name={name},id={id} team={team},position={position} Touchdowns={Touchdowns},PassingInterceptions={PassingInterceptions} FumblesLost={FumblesLost},passingYards={passingYards} passingTouchdowns={passingTouch},reception={reception} receiveYards={receiveYards},receiveTouch={receiveTouch} rushingYards={rushingYards},OffensiveSnapsPlayed={OffensiveSnapsPlayed} DefensiveSnapsPlayed={DefensiveSnapsPlayed},price={price}\n').encode())
```

Comment:
Below is the right spacing for inserting to db

sock.sendall((f'nfl,name={name},id={id},team={team},position={position} Touchdowns={Touchdowns},PassingInterceptions={PassingInterceptions},FumblesLost={FumblesLost},passingYards={passingYards},passingTouchdowns={passingTouch},reception={reception},receiveYards={receiveYards},receiveTouch={receiveTouch},rushingYards={rushingYards},OffensiveSnapsPlayed={OffensiveSnapsPlayed},DefensiveSnapsPlayed={DefensiveSnapsPlayed},price={price}\n').encode(